CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorScreenViewController.swift

```diff
@@ -0,0 +1,241 @@
+        if let header = NoResultHeader().loadNib() as? NoResultHeader {
+            header.headerLabel.adjustsFontForContentSizeCategory = true
+            header.headerLabel.adjustsFontSizeToFitWidth = true
+            header.translatesAutoresizingMaskIntoConstraints = false
+        return header
+        }
+        fatalError("No result header not found")
+    }()
+
+    lazy var buttonsView: ButtonsView = {
+        let view = ButtonsView(
+            firstTitle: NSLocalizedStringPreferredFormat(
+                "ginicapture.error.enterManually",
+                comment: "Enter manually"),
+            secondTitle: NSLocalizedStringPreferredFormat(
+                "ginicapture.error.backToCamera",
+                comment: "Enter manually"))
+        return view
+    }()
+
+    lazy var errorContent: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        return label
+    }()
+
+    let viewModel: BottomButtonsViewModel
+    private let errorType: ErrorType
+    private var buttonsHeightConstraint: NSLayoutConstraint?
+    private var numberOfButtons: Int {
+        return [
+            viewModel.isEnterManuallyHidden(),
+            viewModel.isRetakePressedHidden()
+        ].filter({
+            !$0
+        }).count
+    }
+
+    public init(
+        giniConfiguration: GiniConfiguration,
+        type: ErrorType,
+        viewModel: BottomButtonsViewModel
+    ) {
+        self.giniConfiguration = giniConfiguration
+        self.viewModel = viewModel
+        self.errorType = type
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+
+    func setupView() {
+        title = NSLocalizedStringPreferredFormat(
+            "ginicapture.error.title",
+            comment: "Error screen title")
+        errorHeader.iconImageView.accessibilityLabel = NSLocalizedStringPreferredFormat(
+            "ginicapture.error.title",
+            comment: "Error screen title")
+        errorHeader.headerLabel.text = getErrorTitle(type: errorType)
+        errorHeader.headerLabel.font = giniConfiguration.textStyleFonts[.subheadline]
+        errorHeader.headerLabel.textColor = GiniColor(
+            light: UIColor.GiniCapture.dark1,
+            dark: UIColor.GiniCapture.light1
+        ).uiColor()
+        errorContent.text = getErrorContent(type: errorType)
+        errorContent.textAlignment = .left
+        view.backgroundColor = GiniColor(light: UIColor.GiniCapture.light2, dark: UIColor.GiniCapture.dark2).uiColor()
+        view.addSubview(errorHeader)
+        view.addSubview(errorContent)
+        buttonsView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(buttonsView)
+        errorHeader.backgroundColor = GiniColor(
```

Comment:
[Suggestion] can we move "errorHeader.backgroundColor" up for having all _errorHeader_ configurations together?